scripts/deploy-mainnet-apm.js

```diff
@@ -0,0 +1,68 @@
+const namehash = require('eth-ens-namehash').hash
+const keccak256 = require('js-sha3').keccak_256
+
+const APMRegistryFactory = artifacts.require('APMRegistryFactory')
+const DAOFactory = artifacts.require('DAOFactory')
+const ENS = artifacts.require('ENS')
+
+const owner = '0x2dc1ccf1c83d6760f93adbd66997b56a267ce01a'
+const ens = '0x314159265dD8dbb310642f98f50C066173C1259b'
```

Comment:
Small thing: we could use [ethereum-ens-network-map](https://github.com/danfinlay/ethereum-ens-network-map) for this so we're not just throwing magic strings around.